resources/db/schema/mysql-ddl.sql

```diff
@@ -1087,7 +1087,7 @@ CREATE TABLE user_movies (
   id          INT(16) UNSIGNED               NOT NULL AUTO_INCREMENT,
   user_id INT(16)                        NOT NULL,
   imdbid      MEDIUMINT(7) UNSIGNED ZEROFILL NULL,
-  categoryid  VARCHAR(64)                    NULL DEFAULT NULL,
+  categories_id  VARCHAR(64)                    NULL DEFAULT NULL,
```

Comment:
Really? Are you totally sure about this? I left it alone because it is a varchar and that implies it is being used totally differently to an integer. Everywhere else in the code I've seen a varchar used for category stuff, it was used to store a list of categories, not a single one. So you'd better be 101% sure of this before changing it. Same applies to SQL patch above.